skill-tree /src/test/java/com/RDS/skilltree/EndorsementsIntegrationTests.java

```diff
@@ -0,0 +1,229 @@
+                .body("data[0].endorsementType",anyOf(equalTo("POSITIVE"), equalTo("NEGATIVE")))
+                .body("data[0].endorsersList", hasSize(1))
+                .body("data[0].endorserList[0].endorserId",equalTo("user-2"))
+                .body("data[0].endorserList[0].description",isA(String.class))
+                .body("data[0].endorserList[0].userType",anyOf(equalTo("NORMAL_USER"),equalTo("SUPER_USER"), equalTo("MAVEN")));
+
+
+
+    }
+
+    @Test
+    @Disabled
+    @DisplayName("Fetch all the endorsements given endorsement status")
+    public void testAPIReturnsEndorsementsGivenStatus(){
+        Response response = given()
+                .get("/endorsements?status=PENDING");
+
+        response.then()
+                .statusCode(200)
+                .contentType("application/json")
+                .body("data", hasSize(1))
+                .body("data[0].endorseeId",equalTo("user-1"))
+                .body("data[0].skillName",equalTo("Java"))
+                .body("data[0].status", anyOf(equalTo("APPROVED"), equalTo("REJECTED"), equalTo("PENDING")))
+                .body("data[0].endorsementType",anyOf(equalTo("POSITIVE"), equalTo("NEGATIVE")))
+                .body("data[0].endorsersList", hasSize(1))
+                .body("data[0].endorserList[0].endorserId",equalTo("user-2"))
+                .body("data[0].endorserList[0].description",isA(String.class))
+                .body("data[0].endorserList[0].userType",anyOf(equalTo("NORMAL_USER"),equalTo("SUPER_USER"), equalTo("MAVEN")));
+
+    }
+
+    @Test
+    @Disabled
+    @DisplayName("Return 400 on invalid endorsement status passed")
+    public void testAPIReturns400_OnInvalidStatusPassed(){
+        Response response = given()
+                .queryParam("status","APPROVAL")
+                .get("/endorsements");
+
+        response.then()
+                .statusCode(400)
+                .body("code", equalTo(400))
+                .body("message", equalTo("Invalid status passed"));
+    }
+
+    @Test
+    @Disabled
+    @DisplayName("Return 200 on endorsement creation")
+    public void testAPIReturns200_OnEndorsementCreation() throws JsonProcessingException {
+
+        String requestBody = objectMapper.writeValueAsString(EndorsementCreationRequest.builder()
+                .endorserUserId("u-1")
+                .endorseeUserId("u-2")
+                .skillName("Java")
+                .description("This is endorsed for the skill showcased in Skill tree")
+                .endorsementBehaviour(Behaviour.POSITIVE)
+                .build());
+
+        Response response = given()
+                .contentType("application/json")
+                .body(requestBody)
+                .post("/endorsements");
+        response.then()
+                .statusCode(200)
+                .body("code", equalTo(200))
+                .body("message", equalTo("Created endorsement sucessfully"));
```

Comment:
Nit: its generally a good idea to return the newly created resource details in the POST Response as well.